resources/content/config-content/configscripts/config.sh

```diff
@@ -77,6 +77,9 @@ purge_old()
     done
 }
 
+declare -A archs=( ["x86_64"]="amd64" ["armv7l"]="arm" ["aarch64"]="arm" )
```

Comment:
This is probably not a comprehensive list it would probably be better to do looser checking like 
```shell
case $(uname -m)
arm*)
  ARCH=arm
  ;;
aarch*)
  ARCH=arm
  ;;
*)
  ARCH=x86_64
  ;;
esac
```